e2etest/newe2e_assertions.go

```diff
@@ -240,11 +240,16 @@ func (m MapContains[K, V]) Assert(items ...any) bool {
 		kvPair, ok := v.(KVPair[K, V])
 
 		if !ok {
-			val, ok := v.(V)
-			if !ok {
-				panic("MapContains only accepts KVPair[K,V] or V as items")
+			if val, ok := v.(V); ok {
+				kvPair = m.ValueToKVPair(val)
```

Comment:
Should we remove the condition of v.(V) since we are now using the condition below?

Reply:
I don't think it's bad functionality to have, honestly, even as a fallback.